app/services/personal_key_formatter.rb

```diff
@@ -12,4 +12,9 @@ def self.regexp
   def self.regexp_string
     REGEXP
   end
+
+  def self.code_length
+    # TODO: Put this somewhere else!!
```

Comment:
Do we have a constant somewhere that defines the length of the personal key? I didn't see it, but I could have overlooked it